week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,19 +1,17 @@
 // Fix Functions
 
-// Aim: to understand the change code inside functions 
+// Aim: to understand the change code inside functions
 //
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
-
-
 // 1) mood function does this return `"I am not happy"`
 //    Only make edits inside the function
 
 function mood() {
   let isHappy = true;
 
-  if (isHappy) {
+  if (isHappy === false) {
     return "I am happy";
   } else {
     return "I am not happy";
```

Comment:
Altom your function here is wrong, you don't have to write isHappy === false, because you have assign it true in the let.